README.md

```diff
@@ -1,27 +1,39 @@
-# dojot Deploy - Docker compose
+# Dojot Deploy - Docker Compose
 
-__Attention__: Before running this deployment, it is necessary to define your domain or IP in the [.env](./.env) file in the variable ``DOJOT_DOMAIN_NAME``.
+This deployment option is best suited to development and functional environments. For production environment we recommend to use [Kubernetes](https://github.com/dojot/ansible-dojot).
 
-__Note__ To get completely ready, **healthy**, all services in this `docker-compose` take an average of at least 12 minutes.
```

Comment:
Sugiro manter:
__Note__ To get completely ready, **healthy**, all services in this `docker-compose` take an average of at least 12 minutes.

Reply:
Verifiquei que está na linha 19

Reply:
Esse trecho apenas foi movido para a `linha 29`,  na seção **Usage**.

Reply:
ok